package.json

```diff
@@ -35,6 +35,7 @@
     "redux": "^3.6.0",
     "redux-devtools-extension": "^2.13.2",
     "redux-saga": "^0.16.0",
+    "styled-components": "^4.2.1",
```

Comment:
I never really dig on that but I often read that emotion is faster/lighter and provide same features =)